src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.ts

```diff
@@ -4,6 +4,7 @@ import { EcoNewsService } from '../../services/eco-news.service';
 import { EcoNewsModel } from '../../models/eco-news-model';
 import { Subscription } from 'rxjs';
 import { singleNewsImages } from '../../../../../assets/img/icon/econews/single-news-images';
+import { LocalStorageService } from '../../../../service/localstorage/local-storage.service';
```

Comment:
You can use alias here as well in other imports and write something like:
import { LocalStorageService } from '@service_path/localstorage/local-storage.service';

Reply:
Fixed